<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqVCRToolbar</class>
 <widget class="QToolBar" name="VCRToolbar" >
  <property name="windowTitle" >
   <string>VCR Controls</string>
  </property>
  <property name="orientation" >
   <enum>Qt::Horizontal</enum>
  </property>
  <attribute name="toolBarArea" >
   <enum>TopToolBarArea</enum>
  </attribute>
  <attribute name="toolBarBreak" >
   <bool>false</bool>
  </attribute>
  <action name="actionVCRPlay" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrPlay.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Play</string>
   </property>
  </action>

  <action name="actionVCRPreviousFrame" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrBack.png</iconset>
   </property>
   <property name="text" >
    <string>Pre&amp;vious Frame</string>
   </property>
   <property name="PV_MUST_BE_MASTER" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVCRFirstFrame" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrFirst.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;First Frame</string>
   </property>
  </action>
  <action name="actionVCRNextFrame" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrForward.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Next Frame</string>
   </property>
  </action>
  <action name="actionVCRLastFrame" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrLast.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Last Frame</string>
   </property>
  </action>
  <action name="actionVCRLoop" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../../Components/Resources/pqComponents.qrc" >:/pqWidgets/Icons/pqVcrLoop.png</iconset>
   </property>
   <property name="text" >
    <string>L&amp;oop</string>
   </property>
   <property name="iconText" >
    <string>Loop</string>
   </property>
   <property name="toolTip" >
    <string>Loop</string>
   </property>
   <property name="statusTip" >
    <string>Loop</string>
   </property>
  </action>


  <addaction name="actionVCRFirstFrame" />
  <addaction name="actionVCRPreviousFrame" />
  <addaction name="actionVCRPlay" />
  <addaction name="actionVCRNextFrame" />
  <addaction name="actionVCRLastFrame" />
  <addaction name="actionVCRLoop" />
  <property name="PV_MUST_BE_MASTER" stdset="0">
    <bool>true</bool>
  </property>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
